microservices/forumApi/routes/v1/collaborators/collaborators.js

```diff
@@ -0,0 +1,36 @@
+var db = require('../db/db');
+
+var collaborators = {};
+
+collaborators.subscribe = function(topicId, userId, callback){
+    db.Topic.findById(topicId).exec((err, topic) => {
+        if (err || topic == null) {
+            callback({error:'topic not found'});
+        } else if (topic.contributors.includes (userId)) {
+            callback();
```

Comment:
the later callback (and the former) have the form of err, result with how this one is structured it will be confusing